admin_ui/public/yeep-icon-hamburger.svg

```diff
@@ -0,0 +1 @@
+<svg width="36" height="32" xmlns="http://www.w3.org/2000/svg"><g fill="#FFF" fill-rule="evenodd"><rect y="28" width="36" height="4" rx="2"/><rect y="14" width="36" height="4" rx="2"/><rect width="36" height="4" rx="2"/></g></svg>
```

Comment:
Other than `yeep-logo` that is actually used as an image, all other icons should be reactified, i.e. turned into components. I highly suggest we use https://www.npmjs.com/package/react-icons for conveniency.

Reply:
Happy to play ball to react-icons! There's a weird conflict with our webpack configuration however resulting in a "ReferenceError: module is not defined" when trying to import any of the react-icons.

This is what I tried inside `AsideNav.js`:

```javascript
...
import { FaHome } from 'react-icons/fa';
...
<FaHome />
```

Reply:
ΟΚ, thanks for mentioning. Let me have a look.